index.html

```diff
@@ -5073,7 +5071,7 @@ <h3>Expiration</h3>
 
         <p>
 The <code>expirationDate</code> is expected to be within an expected range
-for the <a>verifier</a>. For example, a <a>verifier</a> can ensure that the
+for the <a>verifier</a>. For example, a <a>verifier</a> can ensure the 
```

Comment:
`can ensure the ` -> `can check that the`

Reply:
Done.